superset/charts/post_processing.py

```diff
@@ -334,7 +334,7 @@ def apply_post_process(
         if query["result_format"] == ChartDataResultFormat.JSON:
             df = pd.DataFrame.from_dict(query["data"])
         elif query["result_format"] == ChartDataResultFormat.CSV:
-            df = pd.read_csv(StringIO(query["data"]))
+            df = pd.read_csv(StringIO(query["data"]), engine="pyarrow")
```

Comment:
The docs mention that

> New in version 1.4.0: The “pyarrow” engine was added as an experimental engine, and some features are unsupported, or may not work correctly, with this engine.

So we should stick to the default to prevent any breaking changes. Any reason why you changed this to pyarrow, other than performance?

Reply:
Only performance. Removed